### PR TITLE
Update docs outdated & Beginning French docs translation

### PR DESCRIPTION
**Pull request by niltied**
Hi @hossainemruz,

The content of this MR below :

1. Update of article "Prepare your site" outdated, mainly block code and hugo version 
2. Addition of the first batch of articles from the doc translated into French, like :
  - Features/index.fr.md
  - Quickstart/index.fr.md
  - Shortcodes/index.fr.md
  - Supports/index.fr.md
  - Getting-started/prepare-site/index.fr.md
  - Getting-started/github_pages/index.fr.md
  - Getting-started/netlify/index.fr.md

### CODE REVIEW

**Comment by niltied** [comment]
Hi @hossainemruz, 

Do you see my MR ? It's seems that no.

**Comment by niltied** [comment]
Oh @hossainemruz, I understand. I'm sorry. Take your time and when you're ready, don't hesitate to give me your feedback.

See you later !

File: README.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
-# Toha Guides
+# Les guides du thème Toha en langue française
```
Comment (hossainemruz) [inline README.md:1]: We can't translate the README file. There can be only one README file. That should be in English. Can you please, make the changes in English?

File: content/posts/supports/index.md [hunk 2]
```diff
@@ -9,14 +9,14 @@ menu:
     weight: 660
 ---
 
-This theme supports adding various support/donation links in your site. Currently, supported support links are:
+Ce thème supporte l'ajout de liens de soutien/donation sur votre site. Actuellement, les liens de soutien supportés sont:
```
Comment (hossainemruz) [inline content/posts/supports/index.md:12]: I suspect you have copy pasted the changes from you French version. Can you make the changes in English?

**Review by hossainemruz — CHANGES_REQUESTED** [review]
Thank you @niltied for this great PR. Almost looks good. Just few small changes necessary. Can you please check the comments and fix them?

File: README.md (hunk 1 above)
Comment (niltied) [reply README.md:1]: Hi @hossainemruz,

My README file is not a translation. I had reset it for my own repo. What I'm going to do, I'm going to put it back to original. No problem.

File: content/posts/supports/index.md (hunk 2 above)
Comment (niltied) [reply content/posts/supports/index.md:12]: You're right too. Yes, i make change in English !

**Review by niltied — COMMENTED** [review]
File updated

**Comment by niltied** [comment]
@hossainemruz ,

I'm not used to making MRs on Github. I reviewed the 2 files. I have updated my repo, but I don't know if I should do anything else afterwards with this MR. Can you check that it's OK for you?

Thank,

**Review by hossainemruz — APPROVED** [review]
Thank you @niltied

**Pull request merged**